roles/pmm/defaults/main.yaml

```diff
@@ -46,44 +46,44 @@ pmm_alertmanager_thr_StaleUploadLog: 3900
 
 pmm_setup_alerting: true
 pmm_delete_alert_rules: false
-pmm_notification_services: ["mysql","mongo","generic","agent","proxysql","postgresql","backup"]
+pmm_notification_services: ["mysql", "mongo", "generic", "agent", "proxysql", "postgresql", "backup"]
 
 pmm_enable_all_alerts_for_service: []
-# pmm_enable_all_alert_for_service: ["mysql","mongo","generic","agent","proxysql","postgresql","backup"]
+# pmm_enable_all_alert_for_service: ["mysql", "mongo", "generic", "agent", "proxysql", "postgresql", "backup"]
 pmm_enable_all_templates_for_service: []
-#pmm_enable_all_templates_for_service: ["mysql","mongo","generic","agent","proxysql","postgresql","backup"]
+# pmm_enable_all_templates_for_service: ["mysql", "mongo", "generic", "agent", "proxysql", "postgresql", "backup"]
 pmm_alerts_for_services: []
 pmm_alert_names: []
 pmm_enabled_alerts: []
 
 pmm_alert_list: []
-#pmm_alert_list:
-#  - name: DiskSpaceWarning
-#    template_name: DiskSpaceWarning
-#    threshold: 30
-#    custom_labels:
-#      label1: '1'
-#      label2: '2'
-#    filters:
-#    - label: env
-#      regexp: staging
-#      type: MATCH
-#    - label: node_name
-#      regexp: db10
-#      type: MISMATCH
-#  - name: DiskSpaceWarning_diff_threshold
-#    template_name: DiskSpaceWarning
-#    threshold: 40
-#    filters:
-#    - label: env
-#      regexp: prod
-#      type: MATCH
-#    - label: node_name
-#      regexp: db11
-#      type: MISMATCH
-#  - name: MySQLInstanceNotAvailable
-#    template_name: MySQLInstanceNotAvailable
-#  - name: VmalertAlertsError
+# pmm_alert_list:
```

Comment:
This is wrong

Reply:
@cezmunsta could you please explain? Is it because it should just be removing `#` to uncomment rather than having to remove both `#` and another space?

Reply:
I disagree with linting of comments. This is inline docs

Reply:
Point taken. I'm fixing it and my ruleset